### PR TITLE
feat: add hetzner

### PR DESCRIPTION
**Pull request by edubacco**

### CODE REVIEW

**Review by pikachuCa — APPROVED** [review]

**Review by xpepper — APPROVED** [review]

File: radar/casavo-tech-radar.json [hunk 1]
```diff
@@ -530,5 +530,12 @@
     "quadrant": "tools",
     "isNew": "FALSE",
     "description": "We use it for asyncronous messaging"
+  },
+  {
+    "name": "Hetzner",
+    "ring": "adopt",
+    "quadrant": "platforms",
+    "isNew": "TRUE",
+    "description": "Many parts of the infrastructure (i.e.: CI, monitoring, internal IT tooling, etc...) are build on hetzner virtual machines, as they are cheaper than AWS"
```
Comment (pferretti) [inline radar/casavo-tech-radar.json:539]: Here I would suggest "hosted" instead of "build" and I would rephrase another part. So:

`Some non-production parts of our infrastructure (i.e.: CI, monitoring, internal IT tooling, etc...) are hosted on Hetzner. The use of Hetzner allows us to reduce and optimise costs.`

**Review by pferretti — CHANGES_REQUESTED** [review]

File: radar/casavo-tech-radar.json [hunk 2]
```diff
@@ -530,5 +530,12 @@
     "quadrant": "tools",
     "isNew": "FALSE",
     "description": "We use it for asyncronous messaging"
+  },
+  {
+    "name": "Hetzner",
+    "ring": "adopt",
```
Comment (pikachuCa) [inline radar/casavo-tech-radar.json:536]: is adopt the right ring ? 

We should put trial no ?

Comment (edubacco) [reply radar/casavo-tech-radar.json:536]: Well I've put "adopt" because even if it was not present in the radar, we've been using it for a long time, and I think that we consider it stable and accepted.
We have vital parts of our infrastructure there (i.e.: all the monitoring stack)

**Review by moebiusmania — APPROVED** [review]

File: radar/casavo-tech-radar.json (hunk 2 above)
Comment (pikachuCa) [reply radar/casavo-tech-radar.json:536]: Yes but there is also AWS on adopt.

Comment (pikachuCa) [reply radar/casavo-tech-radar.json:536]: They kind are the same no ? 
Or you should make it explicit when to use one or the other.

Comment (edubacco) [reply radar/casavo-tech-radar.json:536]: I'm not the right person to answer your question, as I'm not part of the team Infra, and I was not present at the table when they made this choice. What I know is that as AWS costs are ~x10 compared to the hetzner one, we use it "only" for production and client facing workloads, and we try to use hetzner for the rest.

If we want more details, it's better to ask to @giuscri and @pferretti

**Review by giuscri — APPROVED** [review]

File: radar/casavo-tech-radar.json (hunk 2 above)
Comment (pferretti) [reply radar/casavo-tech-radar.json:536]: That's ok to put Hetzner in adopt, since we have been using it for a while.

I understand your point @pikachuCa , AWS and Hetzner have some overlaps. However, in my opinion it is good that these things also emerge in this Tech Radar: the situation resulting from the tech radar does not have to be perfect. On the contrary, it must be a snapshot of what we have today and a basis for understanding which future optimisation and rationalisation actions can bring value in the future.

**Review by pferretti — APPROVED** [review]

File: radar/casavo-tech-radar.json (hunk 2 above)
Comment (xpepper) [reply radar/casavo-tech-radar.json:536]: Even in this case, as we discussed for RabbitMQ vs AWS SQS, what about adding them both?

Comment (pikachuCa) [reply radar/casavo-tech-radar.json:536]: ok ok I was a step too far I think. Like to have only one per category and deal with the overlap right now. But yes. let's merge it.

**Review by pikachuCa — APPROVED** [review]
